CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1
+
+### Vision
+- Changed behaviour on simultaneous `VisionManager` and `VisionReplayManager` instances creation to throwing an exception
```

Comment:
Add `Fixed possible crash on VisionSafety/VisionArManager create/destroy`?

Reply:
Agree, we've put it like 
```
- Fixed a crash that may happen on creating or destroying `VisionARManager` or `VisionSafetyManager`
```

Reply:
done